conmon/conmon.c

```diff
@@ -997,7 +997,9 @@ runtime_exit_cb (G_GNUC_UNUSED GPid pid, int status, G_GNUC_UNUSED gpointer user
 static void
 container_exit_cb (G_GNUC_UNUSED GPid pid, int status, G_GNUC_UNUSED gpointer user_data)
 {
-	ninfof("container %d exited with status %d", pid, status);
+	if (status != 0)
```

Comment:
brackets :smiling_imp:

Reply:
The ifs below it doesn't have brackets :p. Fixed.

Reply:
ahahah, well, we're going to fix that as well :stuck_out_tongue: